client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-main.tsx

```diff
@@ -84,13 +92,27 @@ const ScreenMain = ( {
 							}
 							active={ location.path === NAVIGATOR_PATHS.MAIN_FOOTER }
 						>
-							{ translate( 'Footer' ) }
+							<>
+								{ translate( 'Footer' ) }
+								<PatternCount count={ Number( hasFooter ) } />
+							</>
 						</NavigatorItem>
 					</NavigatorItemGroup>
 				</VStack>
 				{ ! surveyDismissed && <Survey setSurveyDismissed={ setSurveyDismissed } /> }
 			</div>
 			<div className="screen-container__footer">
+				<span className="screen-container__footer-description">
+					{ totalPatternCount > 0 &&
+						translate( 'You’ve added {{strong}}%(count)s{{/strong}} patterns.', {
```

Comment:
Non-blocking. We could improve the translation by:

```
translate( 
  'You’ve added {{strong}}%(count)s{{/strong}} pattern.',
  'You’ve added {{strong}}%(count)s{{/strong}} patterns.',
  {
    count: totalPatternCount,
    args: { count: totalPatternCount },
    components: { 
      string: <strong />,
    },
  }
)
```

Reply:
[nit] We use words like `Select patterns` and `Select styles`. Should we also use `You've selected n patterns`?

Reply:
I'm fine with both. @lucasmendes-design @autumnfjeld Do you have any preference?

Reply:
> We use words like Select patterns and Select styles. Should we also use You've selected n patterns? 

Good point. It makes sense to me as well. Let's use `You've selected n patterns`